doc/Administration-guide/System-components-overview/index.md

```diff
@@ -5,63 +5,73 @@ permalink: /:path/
+Kaa officially supports [MariaDB](https://mariadb.org/) and [PostgreSQL](https://www.postgresql.org/) as embedded SQL databases at the moment.
 
-## Endpoint ##
-A Kaa endpoint is a particular application which uses the Kaa client SDK and resides on a particular connected device. The Kaa endpoint SDK provides functionality for communicating with the Kaa server, managing data locally in the client application, as well as provides integration APIs. The client SDK abstracts the communication protocol, data persistence, and other implementation details that may be specific for any concrete solution based on Kaa.
+#### NoSQL database
 
-## Third-party ##
+NoSQL database instance is used to store endpoint-related data that grows linearly with the amount of managed endpoints.
+Kaa officially supports [Apache Cassandra](http://cassandra.apache.org/) and [MongoDB](https://www.mongodb.com/) as embedded NoSQL database at the moment.
 
-### Zookeeper ###
-Apache ZooKeeper enables highly reliable distributed coordination of Kaa cluster nodes. Each Kaa node push information about connection parameters, enabled services and corresponding services load. Other Kaa nodes use this information in order to get list of their neighbors and communicate with them. Control service uses information about existing Bootstrap services and their connection parameters during SDK generation.
+## High availability (HA)
 
-### SQL database ###
-SQL database instance is used to store metadata about tenants, applications, endpoint groups, etc. This information is shared between endpoints, thus it's volume does not scale and it can be efficiently stored in modern SQL databases. To support high availability of Kaa cluster, SQL database should be also deployed in cluster mode.
+High availability of a Kaa cluster is achieved by deploying both SQL and NoSQL databases in HA mode. 
+Database nodes can be co-located with Kaa nodes on the same physical or virtual machines.
+In order to provide HA of Kaa services, one should deploy topology with at least two Kaa nodes. Each node should act as Control, Operations and Boostrap service simultaniously.
 
-### NoSQL database ###
-NoSQL database instance is used to store information about endpoint profiles, notifications, configurations, etc. The volume of this information scales linearly with amount of endpoints that are managed using particular Kaa cluster instance. NoSQL database nodes can be co-located with Kaa nodes on the same physical or virtual machines. Kaa support Apache Cassandra and MongoDB as a NoSQL database at the moment.
+## Scalability
+
+Multiple nodes with Operations service enabled may be set up in a Kaa cluster for the purpose of horizontal scaling of endpoints requests processing. 
+Multiple nodes with Control service enabled provide horizontal scalability of administration REST API calls processing.
+
+## Load Balancing (LB)
+
+LB task can be decoupled into two subtasks based on originator of requests to Kaa cluster.
```

Comment:
After reading this sentence I have no idea what these tasks are. I think they should be spelled inline.